webapp-django/crashstats/crashstats/management/commands/archivescraper.py

```diff
@@ -208,62 +226,10 @@ def get_json_links(self, path):
 
         return all_json_links
 
-    def scrape_candidates(self, product_name, archive_directory, major_version):
```

Comment:
I moved this below the other method. This method is executed in the parent process and kicks off the worker processes so it makes sense (to me) to have it on the outside rather than buried in the class.